src/java/generator/MapGenerator.java

```diff
@@ -76,10 +77,104 @@ public void save(String folderName, String mapName, SCMap map, long seed) {
 		}
 	}
 
+	private static void interpretArguments(String[] args) {
```

Comment:
Why did you move this entire block? It's probably better to have the generation at the top.
This also makes it impossible to see what parts you changed, please move it back.

Reply:
used intelij to structure code

Reply:
@axel1200 please resolve this so I can merge your additions